<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrimFaceDialog</class>
 <widget class="QWidget" name="TrimFaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trim Face</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="stepLabel">
     <property name="text">
      <string>Step 1 of 4: Select Trimming Curve(s)</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight: bold; font-size: 10pt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="stepDescription">
     <property name="text">
      <string>Select one or more edges or a sketch/wire to be used as trimming tool</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>No curve selected yet</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #0066cc; font-style: italic; padding: 4px;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="curveListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="curveButtonLayout">
     <item>
      <widget class="QPushButton" name="addCurveButton">
       <property name="text">
        <string>Add Curve</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeCurveButton">
       <property name="text">
        <string>Remove Curve</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="selectButton">
     <property name="text">
      <string>Select Face</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="directionFrame">
     <property name="title">
      <string>Projection Direction</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoDirectionCheckbox">
        <property name="text">
         <string>Use automatic direction (face normal)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="manualDirectionLabel">
          <property name="text">
           <string>Manual direction:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="directionCombo">
          <item>
           <property name="text">
            <string>Use face normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Use view direction</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Use custom vector</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewFrame">
     <property name="title">
      <string>Point to Keep</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="instructionsLabel">
        <property name="text">
         <string>Click the button below, then click on the face to select which area to keep after trimming</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pickPointButton">
        <property name="text">
         <string>Pick Point on Face</string>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>28</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="pointLabel">
        <property name="text">
         <string>No point selected</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #0066cc; font-style: italic;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>&lt; Previous</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next &gt;</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishButton">
       <property name="text">
        <string>Finish</string>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>28</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
